### PR TITLE
MDNS Discovery & article removal [HC-1460]

### PR DESCRIPTION
**Pull request by gorillagripcore**
No idea if I understood what mdns-discovery actually does. 

Used these as my main sources: 
https://forum.arduino.cc/t/why-is-mdns-discovery-exe-always-trying-to-access-the-network/1045280/2
https://forum.arduino.cc/t/windows-defender-mdns-discovery/954430/4
https://forum.arduino.cc/t/allow-mdns-discovery-on-macos/1089144

### CODE REVIEW

File: content/Software Support/IDE Settings/About-mdns-discovery.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: "About mdns-discovery"
 id: 4506515275548
 ---
 
-Arduino IDE includes an [mDNS discovery tool](https://github.com/arduino/mdns-discovery) which is used to find available network ports.
+Arduino IDE includes an [mDNS discovery tool](https://github.com/arduino/mdns-discovery) which is used to find available network ports. These ports are essential for enabling communication between the IDE and connected devices, particularly for tasks like over-the-air (OTA) uploads. For example, the Board Manager and Library Manager rely on network ports to detect and manage Arduino boards and libraries connected to the network, ensuring seamless communication and updates. By using mDNS discovery, the Arduino IDE can dynamically detect the presence of compatible devices, making the development process more efficient and streamlined.
```
Comment (seaxwi) [inline content/Software Support/IDE Settings/About-mdns-discovery.md:6]: @per1234 Would you mind having a look at this paragraph? This article sees a fair amount of traffic, and we have received consistent feedback asking for more information about the application and why/if they should allow it.

Comment (Renat0Ribeir0) [inline content/Software Support/IDE Settings/About-mdns-discovery.md:6]:
```suggestion
Arduino IDE includes an [mDNS discovery tool](https://github.com/arduino/mdns-discovery) used to identify the ports of boards that support the Over-The-Air (OTA) feature. This tool helps Arduino IDE communicate with compatible network-connected boards for tasks like OTA uploads. It detects the addition and removal of these ports on the local network, allowing the IDE to keep the list of available ports updated. This ensures that the port listings, such as those found in the _Tools > Port_ menu and the board selector in the IDE, are up-to-date.
```
This version helps explaining that the mDNS tools identifies available network ports, specifically in the context of communication with boards over-the-air.

**Review by Renat0Ribeir0 — CHANGES_REQUESTED** [review]

File: content/Software Support/IDE Settings/About-mdns-discovery.md (hunk 1 above)
Comment (gorillagripcore) [reply content/Software Support/IDE Settings/About-mdns-discovery.md:6]: ok thank you so much 🌞

**Review by Renat0Ribeir0 — APPROVED** [review]

**Review by seaxwi — APPROVED** [review]
LGTM 👍

**Pull request merged**